packages/lib/models/BaseItem.ts

```diff
@@ -967,4 +967,9 @@ export default class BaseItem extends BaseModel {
 		return !!md.match(/^\[.*?\]\(:\/[0-9a-zA-Z]{32}\)$/);
 	}
 
+	public static isLink(text: any) {
+		if (!text) return false;
+		const linkRegex = /^https?:\/\/[^)\s]+$/;
+		return !!text.match(linkRegex);
+	}
```

Comment:
Please move this to `packages/utils/url.ts`.

Also no "any" please. The input is a string. Also if should handle a number of protocols: https://, http://, file://, joplin://

Reply:
Thank you for your review! I've already made the requested changes